server/src/main/java/org/opensearch/index/query/QueryBuilders.java

```diff
@@ -153,18 +153,6 @@ public static IdsQueryBuilder idsQuery() {
         return new IdsQueryBuilder();
     }
 
```

Comment:
The `public static TypeQueryBuilder typeQuery(String type)`  should also be gone, right?

Reply:
yes! not used anywhere